docs/tutorials/ApplicationMigration/testfiles/ro-nginx-service-2.yaml

```diff
@@ -11,6 +11,12 @@ spec:
        name: nginx-service-2
   policy:
     overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  cluster-name: member-1
+        overrideType: Delete
       - clusterSelector:

```

Comment:
why not adding this to the nginx-service-2 itself?

Reply:
it's not a good approach, cause the application admin defines the service, but he/she won't know where the CRP will place the service.

I feel it's better to train the customer to define the annotation in the override, so that they don't need to worry about when/where/if CRP selects this cluster or not.